src/social-network/worker.ts

```diff
@@ -37,20 +40,32 @@ export interface SocialNetworkWorker extends SocialNetworkWorkerAndUI {
     manualVerifyPost?(user: PersonIdentifier, provePost: string): void
 }
 
+export type SocialNetworkWorker = Required<SocialNetworkWorkerDefinition>
+export const getActivatedWorker = getCurrentNetworkWorker
```

Comment:
All workers in the background page are "activated". `getCurrentNetworkWorker` is used to find the coresponding worker by an `Identifier`

Reply:
c7a01283